[W16-4] Recruiting Platform

abwilli opened the pull request:
@charnixon @barisbatuhan @DMGREENHOUSE

lestertj reviewed (commented):
### GitHub project page
- Please update the link at the top of your GitHub page

### Readme Page
- The site doesn't seem to be linked correctly; can you make your repo name uppercase? Comments from this point forth will be based on your .adoc files.
- Try to show more in your mockup; what kind of interface can I expect to see?
- Point your Travis CI build badge to your repo, not S-EDU addressbook

### About Us
- Fill in the components each member will be developing

You will be receiving your UG and DG comments separately.

barisbatuhan committed:
UserGuide

barisbatuhan committed:
UserGuide

barisbatuhan committed:
no message

barisbatuhan committed:
Merge pull request #106 from barisbatuhan/master

UserGuide and Portfolio Updates

barisbatuhan committed:
- Unit tests for FilterCommand and FilterCommandParser are updated

barisbatuhan committed:
-checkstyle errors are fixed

barisbatuhan committed:
no message

barisbatuhan committed:
Merge pull request #107 from barisbatuhan/master

[v1.4] Unit tests for FilterCommand and FilterCommandParser are updated

DanGreenhouse committed:
Fix bug: education, skills, positions, endorsements are now not case sensitive

DanGreenhouse committed:
Rewriting of sort methods to utilise abstraction

DanGreenhouse committed:
Updates to user guide specifying lower case sorting ability for endorsements, positions and skills

DanGreenhouse committed:
Addition of position and endorsement tag attributes to persons in the PersonBuilder for testing purposes

abwilli committed:
Merge pull request #8 from cs2103-ay1819s2-w16-4/master

local merge

abwilli committed:
Merge pull request #108 from abwilli/master

Endorse Command updates

abwilli committed:
Enhanced Endorse Command with Clear function

abwilli committed:
Merge pull request #109 from abwilli/master

Checkstyle issues

abwilli committed:
Checkstyle issues

DanGreenhouse committed:
Additional methods to person class: firstNameToString and getPositionString

DanGreenhouse committed:
Changes to browser panel to include linked in page. 5 different searching methods are used and all temporarily kept here for the purpose of the upcoming deadline

DanGreenhouse committed:
Changes to the tests to allow for different tags. In general previous tags were changed to skills and there is now room for seperate skill, position and endorsement tags. These are all utilised by the PersonBuilder.

DanGreenhouse committed:
Typical persons updated to include the new skill, endorsement and position fields. FilterCommand tests are updated accordingly

DanGreenhouse committed:
Update to the AddCommand error message to suit the new product

DanGreenhouse committed:
Added to SortCommandTest to test the new sorting methods (by skills, positions, endorsements, skill number, position number, endorsement number)

DanGreenhouse committed:
Re-include default webpage for testing purposes

DanGreenhouse committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w16-4/main

DanGreenhouse committed:
Fix SkillsTag calls (sometimes the type was not being recorded while other times it was)

DanGreenhouse committed:
Check Style Errors

DanGreenhouse committed:
Check Style Changes

DMGREENHOUSE committed:
Merge pull request #110 from DMGREENHOUSE/master

mid v1.4

barisbatuhan committed:
Merge pull request #12 from cs2103-ay1819s2-w16-4/master

Updates are taken

DanGreenhouse committed:
Bug fix for adapted tag and updates to browser panel and test to include the linked in panel

DanGreenhouse committed:
Bug fix

DanGreenhouse committed:
PersonListPanelTest Test fix:
the issue was with the new tags, so additional persons with no tags are added and used for the test

DanGreenhouse committed:
Test fixes: JsonAdaptedPersonTest -> solution: use a person without tags to circumvent issue
Update Json typical persons to match the new typical persons

DanGreenhouse committed:
Check Style error alterations

DanGreenhouse committed:
More check style error corrections

DMGREENHOUSE committed:
Merge pull request #112 from DMGREENHOUSE/master

Test Updates

barisbatuhan committed:
Merge pull request #13 from cs2103-ay1819s2-w16-4/master

Updates are taken

barisbatuhan committed:
- Failing Edit tests are fixed. Since Editing process is changed (Before: with using filteredPersonsList from model class / after: using persons list from AddressBook class), the unnecessary tests are excluded

barisbatuhan committed:
Merge pull request #113 from barisbatuhan/master

[v1.4] All failing tests are fixed except systemtests

barisbatuhan committed:
checkStyleFixes

barisbatuhan committed:
Merge pull request #114 from barisbatuhan/master

checkStyleFixes

barisbatuhan committed:
- For filtering, new tests are added

barisbatuhan committed:
Merge pull request #115 from barisbatuhan/master

For filtering, new tests are added, old tests are rewritten

barisbatuhan committed:
Systemtest changes

barisbatuhan committed:
Merge pull request #116 from barisbatuhan/master

Systemtest changes

barisbatuhan committed:
Checkstyle fixes

barisbatuhan committed:
Merge pull request #117 from barisbatuhan/master

Checkstyle fixes

DanGreenhouse committed:
Secondary sorting for persons of matching surnames

charnixon committed:
Merge branch 'master' into master

DanGreenhouse committed:
Rewriting of sort methods to include a SortMethod Interface.

DanGreenhouse committed:
Implementation of 'Intelligent Sorting'.
The sort methods go through a second round of sorting if they returned matching person paramaters on the first sorting method. The choice of this secondary sorting method is chosen to be most useful for the user (e.g. if skill number is the first sort, then the second sort shall be by skills alphabetically)

DanGreenhouse committed:
Sort command test updated as per new sorting methods

barisbatuhan committed:
Merge pull request #14 from cs2103-ay1819s2-w16-4/master

Updates are taken

barisbatuhan committed:
Filter command is updated
Unit tests and Integration tests are written for Filter Command

barisbatuhan committed:
checkStyle Errors

barisbatuhan committed:
checkStyle Fixes

barisbatuhan committed:
checkstyle

barisbatuhan committed:
Filter Command Integration and Unit Tests

abwilli committed:
Merge pull request #9 from cs2103-ay1819s2-w16-4/master

local update

abwilli committed:
user guide and developer guide changes

abwilli committed:
Merge branch 'master' of https://github.com/abwilli/main

abwilli committed:
Endorse command test

abwilli committed:
checkstyle tests

abwilli committed:
check style fixes again

abwilli committed:
checkstyle again

abwilli committed:
kill me check style

abwilli committed:
test error

barisbatuhan committed:
CV showing is implemented, Filtering Integration tests are finished, Filter command is updated

barisbatuhan committed:
checkstyle

barisbatuhan committed:
Merge pull request #121 from barisbatuhan/master

CV showing is implemented, Changes in filtering and test cases

DanGreenhouse committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w16-4/main

abwilli committed:
test passing

abwilli committed:
Merge pull request #120 from abwilli/master

Endorse command test

barisbatuhan committed:
Merge pull request #15 from cs2103-ay1819s2-w16-4/master

Updates are taken

abwilli committed:
Merge pull request #10 from cs2103-ay1819s2-w16-4/master

local update

abwilli committed:
Merge pull request #122 from abwilli/master

Endorse Bug Fix

barisbatuhan committed:
Merge pull request #16 from cs2103-ay1819s2-w16-4/master

Updates are taken

abwilli committed:
endorse/filtering bug fix

barisbatuhan committed:
Codacy Badge

barisbatuhan committed:
Codacy Badge

barisbatuhan committed:
Coveralls

barisbatuhan committed:
Coveralls

barisbatuhan committed:
Coveralls

barisbatuhan committed:
Merge pull request #123 from barisbatuhan/master

Coveralls and Codacy Implementation

barisbatuhan committed:
Ui image is changed

barisbatuhan committed:
Delete UI.png

barisbatuhan committed:
Add files via upload

barisbatuhan committed:
Merge pull request #124 from barisbatuhan/master

UI image is updated

abwilli committed:
endorse clear test

abwilli committed:
check style

abwilli committed:
Merge pull request #125 from abwilli/master

endorse clear test

barisbatuhan committed:
Image Addition

barisbatuhan committed:
Merge branch 'master' of https://github.com/barisbatuhan/main

barisbatuhan committed:
Merge pull request #126 from barisbatuhan/master

Image is converted to png from jpg

barisbatuhan committed:
README file informations are updated

barisbatuhan committed:
Merge pull request #127 from barisbatuhan/master

README file informations are updated

abwilli committed:
endorse duplicate test

abwilli committed:
Merge pull request #128 from abwilli/master

endorse duplicate test

abwilli committed:
more endorse command tests

barisbatuhan committed:
UserGuide Updated

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
UserGuide Update with CV information

barisbatuhan committed:
no message

barisbatuhan committed:
UserGuide Updates

barisbatuhan committed:
Merge pull request #129 from barisbatuhan/master

UserGuide is updated

abwilli committed:
edit command test

abwilli committed:
Merge pull request #130 from abwilli/master

Edit Command Test

charnixon committed:
addition of the Degree and usability in ui

charnixon committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w16-4/main

barisbatuhan committed:
Merge pull request #17 from CS2103-AY1819S2-W16-4/master

Updates are taken

charnixon committed:
style

charnixon committed:
Merge pull request #131 from charnixon/master

style

charnixon committed:
style fixes

charnixon committed:
Merge pull request #132 from charnixon/master

style fixes

DanGreenhouse committed:
Updates to sort method and initial blind implementation of degree sort.
Commited now to retieve Charlotte's latest PR

DanGreenhouse committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w16-4/main

barisbatuhan committed:
- Tests are Fixed
- UI image is updated

barisbatuhan committed:
Merge pull request #133 from barisbatuhan/master

Failing tests are fixed, UI image is updated

barisbatuhan committed:
checkStyleErrors

barisbatuhan committed:
Merge branch 'master' into master

barisbatuhan committed:
Merge pull request #134 from barisbatuhan/master

checkStyleErrors

barisbatuhan committed:
checkStyle Errors

barisbatuhan committed:
Merge branch 'master' of https://github.com/barisbatuhan/main

barisbatuhan committed:
Merge pull request #135 from barisbatuhan/master

checkStyle Fixes

barisbatuhan committed:
checkStyle

barisbatuhan committed:
Merge pull request #136 from barisbatuhan/master

checkStyle

charnixon committed:
correctForm for gpa

charnixon committed:
reassigned input parameter to gpa

charnixon committed:
Merge pull request #137 from charnixon/master

correctForm for gpa

barisbatuhan committed:
Merge pull request #18 from CS2103-AY1819S2-W16-4/master

Updates are taken

barisbatuhan committed:
checkStyle

barisbatuhan committed:
checkStyle test

barisbatuhan committed:
checkStyle Test fixes

barisbatuhan committed:
Merge pull request #138 from barisbatuhan/master

checkStyle Main fix

charnixon committed:
Merge pull request #1 from CS2103-AY1819S2-W16-4/master

local changes

barisbatuhan committed:
Degree filtering is added, test cases are arranged according to that

barisbatuhan committed:
checkStyle errors

barisbatuhan committed:
Code Quality Arrangements

barisbatuhan committed:
Process fix

barisbatuhan committed:
Merge pull request #139 from barisbatuhan/master

Degree filtering is added, test cases are arranged according to that

barisbatuhan committed:
System tests are fixed

barisbatuhan committed:
CHECKsTYLE TEST FIXES

barisbatuhan committed:
Merge pull request #140 from barisbatuhan/master

System tests are fixed

DanGreenhouse committed:
Full implementation of sort degree

barisbatuhan committed:
Document updates

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
Merge pull request #141 from barisbatuhan/master

Documentation Updates

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

DanGreenhouse committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w16-4/main

Also changes to sort tests

DanGreenhouse committed:
Checkstyle

DanGreenhouse committed:
Remove print statement

DanGreenhouse committed:
Checkstyle

DanGreenhouse committed:
FilterCommandIntegrationTest change to new sorting method interface

DanGreenhouse committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w16-4/main

DMGREENHOUSE committed:
Merge pull request #4 from CS2103-AY1819S2-W16-4/master

Update to latest

DanGreenhouse committed:
Merge branch 'master' of https://github.com/DMGREENHOUSE/addressbook-level4

DanGreenhouse committed:
Conflict res

DanGreenhouse committed:
Conflict res

DanGreenhouse committed:
filter sort integration conflict

DanGreenhouse committed:
Filter -sort-reverse testing corrected and passing

DanGreenhouse committed:
Addition of sort degree test

DanGreenhouse committed:
Check Style

DanGreenhouse committed:
Check Style

DanGreenhouse committed:
Code Quality update

DanGreenhouse committed:
Code quality update

DanGreenhouse committed:
Coding standard change and PPP update

DanGreenhouse committed:
Removal of user guide/ developer guide links

DMGREENHOUSE committed:
Merge pull request #142 from DMGREENHOUSE/master

Implementation of Degree Sort and Sequential Sorting Methods

barisbatuhan committed:
Merge pull request #19 from CS2103-AY1819S2-W16-4/master

Updates taken

DanGreenhouse committed:
User Guide update

DanGreenhouse committed:
Style checking

barisbatuhan committed:
Developer Guide Update

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
no message

barisbatuhan committed:
Merge pull request #143 from barisbatuhan/master

Developer Guide and Portfolio is finalized.

charnixon committed:
Merge pull request #2 from CS2103-AY1819S2-W16-4/master

local changes

barisbatuhan committed:
no message

barisbatuhan committed:
Merge pull request #149 from barisbatuhan/master

no message

barisbatuhan committed:
no message

barisbatuhan committed:
Merge pull request #150 from barisbatuhan/master

UserGuide and barisbatuhan Portfolio is updated

barisbatuhan committed:
no message

barisbatuhan committed:
Merge pull request #151 from barisbatuhan/master

Document Updates

barisbatuhan committed:
no message

barisbatuhan committed:
Merge pull request #152 from barisbatuhan/master

Developer Guide Update

barisbatuhan committed:
no message

barisbatuhan committed:
Merge pull request #153 from barisbatuhan/master

Developer Guide Tag Update

barisbatuhan committed:
no message

barisbatuhan committed:
Merge pull request #154 from barisbatuhan/master

Developer Guide Update

barisbatuhan committed:
no message

barisbatuhan committed:
Merge pull request #155 from barisbatuhan/master

Developer Guide Update

barisbatuhan committed:
no message

barisbatuhan committed:
Merge pull request #156 from barisbatuhan/master

Sample Data at Beginning is Improved

DanGreenhouse committed:
Documentation update

charnixon committed:
Merge pull request #3 from CS2103-AY1819S2-W16-4/master

local changes

charnixon committed:
Model Diagrams

charnixon committed:
Merge pull request #157 from charnixon/master

Model Diagrams

abwilli committed:
Added Endorse Command Diagram

abwilli committed:
Developer and User Guide changes

abwilli committed:
Merge branch 'master' into master

barisbatuhan committed:
Update DeveloperGuide.adoc

barisbatuhan committed:
Developer Guide Prerequisites Update

abwilli committed:
Merge pull request #158 from abwilli/master

Developer and User Guide changes

barisbatuhan committed:
Merge pull request #20 from CS2103-AY1819S2-W16-4/master

Updates are taken

barisbatuhan committed:
no message

barisbatuhan committed:
Merge pull request #159 from barisbatuhan/master

Developer Guide Fix

DanGreenhouse committed:
Final Developer Guide

DanGreenhouse committed:
Merge branch 'master' of https://github.com/cs2103-ay1819s2-w16-4/main

abwilli committed:
Merge pull request #11 from CS2103-AY1819S2-W16-4/master

local updates

abwilli committed:
developer changes

abwilli committed:
Merge branch 'master' of https://github.com/abwilli/main

DanGreenhouse committed:
PPP update

DanGreenhouse committed:
Checkstyle change

abwilli committed:
Typos and PPP

DMGREENHOUSE committed:
Merge pull request #160 from DMGREENHOUSE/master

Final PR for v1.4, documentation updates

charnixon committed:
PPP

barisbatuhan committed:
Merge pull request #21 from CS2103-AY1819S2-W16-4/master

Updates are taken

charnixon committed:
Merge pull request #161 from charnixon/master

PPP

barisbatuhan committed:
no message

barisbatuhan committed:
Merge pull request #162 from barisbatuhan/master

Tag in a wrong place problem fix in Developer Guide

abwilli committed:
PPP updates

abwilli committed:
Merge pull request #163 from abwilli/master

Final v1.4 changes

charnixon committed:
Fixes to the PPP

charnixon committed:
Merge pull request #164 from charnixon/master

Fixes to the PPP
